tests/integration/test_pipeline_steps.py

```diff
@@ -36,6 +35,16 @@ def temp_workdir_with_contigs():
         yield workdir
 
 
+@pytest.fixture
+def overlapping_contig(temp_workdir_with_contigs):
+    # Create an additional contig that overlaps with seed but doesn't contain it
+    megahit_dir = temp_workdir_with_contigs / "megahit_out_iter1-1"
+    contigs_path = megahit_dir / "final.contigs.fa"
+    # Add a contig with overlap to existing contigs
+    with open(contigs_path, "a") as f:
+        f.write(">contig6\nGCGCGTAATATAAAGGCC\n")  # Contains seed and overlaps with contig5
```

Comment:
I think I'm confused here, though very possibly because I don't understand pytest magics.

The comment here says `contains seed`, though the comment 5 lines up says the opposite. But which is right? Where is the seed defined?

Reply:
Ah I see that a test like `test_subset_contigs_preserves_seed_orientation_for_overlaps` which uses this fixture can define a seed internally. Though in that case, should this fixture refer to seeds at all?

Reply:
Sorry, yep, I pulled this logic out from a pre-existing test (test_subset_contigs_with_overlaps) whose logic I wanted to reuse, but I copied the comments too, which don't make sense in context. Updating